www/web-proxy-sso/src/opnsense/mvc/app/models/OPNsense/ProxySSO/ProxySSO.php

```diff
@@ -3,7 +3,15 @@
 namespace OPNsense\ProxySSO;
 
 use OPNsense\Base\BaseModel;
+use OPNsense\Core\Config;
 
 class ProxySSO extends BaseModel
 {
+	protected function init()
+	{
+		if($this->KerberosHostName == "") {
+			$hostname = (string)Config::getInstance()->object()->system->hostname;
+			$this->KerberosHostName = substr(strtoupper($hostname), 0, 13) . '-K';
```

Comment:
Or set to required N and fall back here...

Reply:
OK

Reply:
Done.